ui/src/app/components/ct-available-banner-maybe.spec.tsx

```diff
@@ -135,28 +131,22 @@ describe('CTAvailableBannerMaybe', () => {
   const userIsNew = () => {
     updateProfile({ newFirstSignIn: TIME3 });
   };
-  const darActive = () => {
-    window.location.pathname = DATA_ACCESS_REQUIREMENTS_PATH;
```

Comment:
this test may have been incorrect but inappropriately passing because we did not `await` the wrapper update.  Now we set this at the react-router level.